sys/linux/init_vusb.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package linux
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/google/syzkaller/prog"
+)
+
+const (
+	USB_DEVICE_ID_MATCH_VENDOR       = 0x0001
```

Comment:
For consecutive bits less error prone to use and easier to read:
```
const (
	USB_DEVICE_ID_MATCH_VENDOR       = 1 << iota
        USB_DEVICE_ID_MATCH_PRODUCT
        ...
)
```

Reply:
Done.